client/src/components/ActivitatiSecundare.js

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import cs2 from "../assets/img/webp/Lol-CS/cs2.webp";
+import logo_cs from "../assets/img/webp/Lol-CS/logo_cs.webp";
+import logo_lol from "../assets/img/webp/Lol-CS/logo_lol.webp";
+import lol from "../assets/img/webp/Lol-CS/lol.webp";
+
+import tft from "../assets/img/webp/activitatisecundare/tft_fara_logo.webp";
+import ea from "../assets/img/webp/activitatisecundare/ea_rosu.webp";
+import hearth from "../assets/img/webp/activitatisecundare/hearthstone.webp";
+import mortal from "../assets/img/webp/activitatisecundare/mortal.webp";
+import clash from "../assets/img/webp/activitatisecundare/clash-royale.webp";
+import logo_tft from "../assets/img/webp/activitatisecundare/tft_logo.webp";
+import bg_tft from "../assets/img/webp/activitatisecundare/teamfight.webp";
+
+export default function JocuriSecundare()
+{
+    return (
+    
+        <ul id="cards">
+            <li className="lol">
+                <a target="_blank" href="https://support.riotgames.com/hc/en-us">
+                    <div className="w-full h-full"><img src={bg_tft}/></div>
```

Comment:
all images should also contain the alt attribute 

https://www.w3schools.com/tags/att_img_alt.asp